### PR TITLE
Use pegs instead of re for file match

### PR DESCRIPTION
**Pull request by kdeme**

### CODE REVIEW

**Comment by arnetheduck** [comment]
ugh, this is a bit sad, changing from a well-known grammar to something that essentially nobody (wants to) uses, just because we can't get our build together

**Comment by kdeme** [comment]
well, I changed it not just for that (it was the trigger yes), but because I thought it was the nim philosophy to use pure as much as possible. Or am I wrong about that?

Next to that, pegs is already used for other things, so one dep less...

**Comment by kdeme** [comment]
By the way, I do agree about the well-know grammar thing. It is just that nim seems to favor pegs.

**Comment by stefantalpalaru** [comment]
Why not use `file.endsWith(".test")`?

**Comment by kdeme** [comment]
sure, even better

**Comment by cheatfate** [comment]
@arnetheduck using regexp engine for capturing  `*` and `?` is like using Electron to show message box.

File: tests/testrunner.nim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import os, osproc, re, strutils, terminal, times, pegs
+import os, osproc, strutils, terminal, times, pegs
```
Comment (arnetheduck) [inline tests/testrunner.nim]: can remove the pegs import now

Comment (kdeme) [reply tests/testrunner.nim]: pegs are still used: https://github.com/status-im/nim-chronicles/blob/191b09832f426d48ec23603687f9b661ca7b39a8/tests/testrunner.nim#L107

**Pull request merged**